workspaces/orchestrator/plugins/orchestrator-backend/src/service/DevModeService.ts

```diff
@@ -197,7 +208,7 @@ export class DevModeService {
     const persistencePath =
       config.getOptionalString(
         'orchestrator.sonataFlowService.persistence.path',
-      ) ?? DEFAULT_SONATAFLOW_PERSISTENCE_PATH;
+      ) ?? './.devModeTemp/db_persistence';
```

Comment:
I'd like to keep the default as variable instead of using an hardcode path. Is it possible?

Reply:
@gciavarrini Great idea, will do that change.